soql-analyzer/src/main/scala/com/socrata/soql/analyzer2/package.scala

```diff
@@ -1,18 +1,65 @@
 package com.socrata.soql
 
+import scala.reflect.ClassTag
+import scala.util.parsing.input.{Position, NoPosition}
+
+import com.rojoma.json.v3.ast.{JNull, JString, JValue}
+import com.rojoma.json.v3.codec.{JsonEncode, JsonDecode, DecodeError}
+import com.rojoma.json.v3.matcher.{Variable, PObject}
+import com.rojoma.json.v3.util.SimpleHierarchyCodecBuilder
+
+import com.socrata.soql.parsing.SoQLPosition
+import com.socrata.soql.util.{SoQLErrorCodec, SoQLErrorEncode, SoQLErrorDecode, EncodedError}
+
 package object analyzer2 {
-  // type alias for referring to any generic soql error since Scala
-  // doesn't have type-parameter defaulting.
-  type SoQLError[+RNS] = SoQLAnalyzerError[RNS, SoQLAnalyzerError.Payload]
+  type AliasAnalysisError[+RNS] = SoQLAnalyzerError.AliasAnalysisError[RNS]
+  type TypecheckError[+RNS] = SoQLAnalyzerError.TypecheckError[RNS]
+
+  // private[analyzer2] implicit class AugCodec[T <: AnyRef](private val shc: SimpleHierarchyCodecBuilder[T]) extends AnyVal {
```

Comment:
Keeping this around for any particular reason?

Reply:
Nope!  It's obsolete and I just forgot to clean it up.